frontend/src/components/PreviewReview/PreviewReviewTest.tsx

```diff
@@ -0,0 +1,30 @@
+import { Button, useDisclosure } from "@chakra-ui/react";
+import React, { useEffect, useState } from "react";
+
+import reviewAPIClient from "../../APIClients/ReviewAPIClient";
+import { ReviewResponse } from "../../types/ReviewTypes";
+import PreviewReviewModal from "./PreviewReviewModal";
+
+/**
+ * A test page for testing the PreviewReviewModal
+ * TODO: Remove once PreviewReviewModal is integrated
+ */
+const PreviewReviewTest = (): React.ReactElement => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [data, setData] = useState<ReviewResponse[]>([]);
+
+  useEffect(() => {
+    reviewAPIClient.getReviews().then((allReviews: ReviewResponse[]) => {
```

Comment:
If this component is just to test the modal, it shouldn't rely on an API call (ie might be better to make the data hardcoded)